foundations/foundation-log/src/main/java/org/apache/servicecomb/foundation/log/LogConfig.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.servicecomb.foundation.log;
+
+import com.netflix.config.DynamicPropertyFactory;
+
+public class LogConfig {
+
+    private static final String SERVER_BASE = "servicecomb.accesslog.";
+
+    private static final String CLIENT_BASE = "servicecomb.outlog.";
```

Comment:
`servervicomb.request.log`  or `servicecomb.accesslog.request` is better, see 'https://sling.apache.org/documentation/development/client-request-logging.html' for reference.

Reply:
ok, **servervicomb.request.log**  is better